wordpress-mobile/WordPress-iOS
# Resolved Alpha & Internal signing issues for notification extensions

## Description
Corrected a couple of issues identified with builds of the app with the notification content & service extensions.

## To test
- Branch should build and existing tests should pass.
- It should also now be possible to build the `WordPress`, `WordPress Alpha` & `WordPress Internal` targets without issue. Notably, Building the `WordPress Alpha` scheme now uses the `Debug` _Build Configuration_ rather than the `Release-Alpha` configuration.
